### ISSUE TITLE
J7 class for “Indicator” 

### ISSUE THREAD

**Issue by xgmachina** [USGCRP/gcis-ontology#105]
Please create a term for “Indicator.”  
See the page accessible at: http://www.globalchange.gov/browse/indicators

**Comment by justgo129** [comment]
@rewolfe I'm content to use the definition on the aforementioned webpage:
"Indicators are observations or calculations that can be used to track conditions and trends. "

Please confirm that this is acceptable.

**Comment by rewolfe** [comment]
Good definition. +1

On Mon, Jul 27, 2015 at 9:55 PM, justgo129 notifications@github.com wrote:

> @rewolfe https://github.com/rewolfe I'm content to use the definition
> on the aforementioned webpage:
> "Indicators are observations or calculations that can be used to track
> conditions and trends. "
> 
> Please confirm that this is acceptable.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-125405039
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
Thanks, @rewolfe.  Would the domain be a "report" or would it be blank?

**Comment by rewolfe** [comment]
@justgo129 - I think so

On Tue, Jul 28, 2015 at 11:03 AM, justgo129 notifications@github.com
wrote:

> Thanks, @rewolfe https://github.com/rewolfe. Would the domain be a
> "report" or would it be blank?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-125640662
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by rewolfe** [comment]
@justgo129 - actually, I meant that I think the domain should be a report

On Tue, Jul 28, 2015 at 11:07 AM, Robert Wolfe rewolfe@usgcrp.gov wrote:

> @justgo129 - I think so
> 
> On Tue, Jul 28, 2015 at 11:03 AM, justgo129 notifications@github.com
> wrote:
> 
> > Thanks, @rewolfe https://github.com/rewolfe. Would the domain be a
> > "report" or would it be blank?
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-125640662
> > .
> 
> ## 
> 
> Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by xgmachina** [comment]
@justgo129 Oh, originally I was thinking you suggest a class for 'Indicator'. Or, can we create a pair - `gcis:Indicator` as a class and `gcis:hasIndicator` as a property?

**Comment by zednis** [comment]
Let's work through an example to determine if indicator should be a class or just a property.

**Comment by zednis** [comment]
Also, TW previously did some modeling of indicators related to the DPSIR conceptual framework.  http://tw.rpi.edu/media/2012/12/01/a279/AGUFM2012-ED41A-0666-PW.pptx

This may be useful to review.

**Comment by justgo129** [comment]
We won't need gcis:hasIndicator.  If anything it would be a subclass of report, or a subclass of a document, An example is:
https://data.globalchange.gov/report/indicator-start-of-spring

No need to worry about this too much since it's not really a high priority for now.

**Comment by xgmachina** [comment]
From that example it seems an indicator is a kind of report.

**Comment by rewolfe** [comment]
We've discussed making it a subclass of publication (like book, report,
etc.), instead of a kind of report.

On Tue, Jul 28, 2015 at 2:39 PM, Marshall.Ma notifications@github.com
wrote:

> From that example it seems an indicator is a kind of report.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-125708512
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
@rewolfe do you advise we take further action at this time or hold off?

**Comment by rewolfe** [comment]
@justgo129 - There are a couple of clear definitions here:
https://en.wikipedia.org/wiki/Environmental_indicator

USGCRP uses the term "Indicator" as a type of web publication.  Based on
the way USGCRP is using the term, I would change the definition to say:

"An Indicator is a published set of observations or calculations that is
used for understanding conditions and trends."

On Tue, Aug 25, 2015 at 9:26 AM, justgo129 notifications@github.com wrote:

> @rewolfe https://github.com/rewolfe do you advise we take further
> action at this time or hold off?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-134584724
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
Sure, and for domain/range, are we asserting it to be a type of report or its own publication class?

**Comment by rewolfe** [comment]
I know that I've been vacillating between "type of report" and "own
publication class".  I think the advantage of making it a type of report is
that we can associate figures, tables and references to it if it is a
sub-class of report.  However, the downside is that we don't really want to
associate it with chapters or findings.  I'm looking at our data model at
http://data.globalchange.gov/resources and trying to understand how it
would fit in with our definitions there.

On Tue, Aug 25, 2015 at 10:17 AM, justgo129 notifications@github.com
wrote:

> Sure, and for domain/range, are we asserting it to be a type of report or
> its own publication class?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-134600681
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
No sweat.  I'll keep this ticket open in the meantime.

**Comment by rewolfe** [comment]
@bduggan - Could you weigh in on this?

On Tue, Aug 25, 2015 at 10:42 AM, justgo129 notifications@github.com
wrote:

> No sweat. I'll keep this ticket open in the meantime.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-134609181
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by bduggan** [comment]
I think @aulenbac may have ideas. The current relational representation is just a subclass of a report, but the concept is broader and includes, I would say, something about sequences of measurements or observations and possibly a visualization of them.

**Comment by zednis** [comment]
The definition we have been using suggests that datasets can also be indicators.

"Indicators are observations or calculations that can be used to track conditions and trends. "

I am not sure we want to define indicator so narrowly that it is a subclass of report.  If we do though, we can still assert that datasets or other non-report publications are indicators.  We have not modeled report as disjoint from other publication classes or dataset, so it is not technically inconsistent to say:

```
ex:dataset1
  a gcis:Dataset, gcis:Report, gcis:Indicator ;
  rdfs:comment "example dataset that is also an indicator" .
```

what this example is missing is a description of what this indicator is an indicator ~of~.

In the example https://data.globalchange.gov/report/indicator-start-of-spring, the report is an indicator of "start of spring".  I think we should have a property to allow us to assert what concept the indicator is an indicator of.

```
gcis:Indicator
  a ow:Class ;
  rdfs:subClassOf gcis:Publication ;
  rdfs:label "Indicator" ;
  rdfs:comment "Indicators are observations or calculations that can be used to track conditions and trends" .

gcis:indicatorOf
  a owl:ObjectProperty ;
  rdfs:label "indicator of" ;
  rdfs:domain gcis:Indicator ;
  rdfs:range skos:Concept .
```

so the indicator-start-of-spring example would look like this.

```
<http://data.globalchange.gov/report/indicator-start-of-spring>
  a gcis:Report, gcis:Indicator ;
  gcis:indicatorOf <http://data.globalchange.gov/concept/start-of-spring>
```

We might not currently have the indicator concepts in the DB to support the `gcis:indicatorOf` property - but I think it would be a good aspirational goal to support this in the future as it the concepts could be used in faceted search.

@bduggan I think this could align with the CDI human health initiative.

**Comment by justgo129** [comment]
@rewolfe @bduggan what do you think?

**Comment by rewolfe** [comment]
An indicator should be a type (subtype) of report.  It has the similar
types of relationships as reports for resources such as figures, tables,
datasets and references.  The main thing it is missing is chapters, but a
report is not required to have chapters.

I don't really understand the need for "indicatorOf".  Suppose we look at
our definition of report:

"An account given of a particular matter, especially in the form of an
official document, after thorough investigation or consideration by an
appointed person or body."

If we follow an "indicatorOf" approach, we could decide to have a property
"gcis:accountOf" to say what "particular matter" the report is an
accountOf.  I don't find that very useful.  It seems that "accountOf" is
already pretty well covered through the report's "dcterms:title" and
"dcterms:description" properties.

On Tue, Sep 22, 2015 at 3:15 PM, justgo129 notifications@github.com wrote:

> @rewolfe https://github.com/rewolfe @bduggan
> https://github.com/bduggan what do you think?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/105#issuecomment-142389817
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
:+1: to the suggestion of @rewolfe.

**Comment by justgo129** [comment]
@xgmachina @zednis please feel free to proceed per the comment of @rewolfe.

**Comment by justgo129** [comment]
Closed #105 on behalf of merged #161.
